src/components/pages/add-reservation/ReserveForm.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable */
+import React from 'react';
+import { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate, useLocation } from 'react-router-dom';
+import { selectRoomsList } from '../../../apis/v1/rooms';
+import { addNewReserve } from '../../../redux/features/reservations/mine';
+function ReserveForm() {
```

Comment:
- [ ] You are using the "function" keyword for some of your functions and components.  Please, refactor those functions to ES6 arrow functions.

## Explanation

All over the software industry, the "function" keyword has been soft deprecated in favor of ES6 functions.

## Tips and extra help

Just do a quick search in Vscode for the word "function" so you can find them all, and change each one to ES6 arrow functions.  Don't worry, it should take no more than 5-10 minutes.  

That way,  your app will comply with the latest standards 👍